Do not disable new button and search while loading table's content (#1054)
vtex/styleguide: CHANGELOG.md

## [Unreleased]

## Changed

> - Do not disable newLine button and search on Table's toolbar while loading

Reviewer: ```suggestion
- Keep newLine button and search enabled on Table's toolbar while loading
```
